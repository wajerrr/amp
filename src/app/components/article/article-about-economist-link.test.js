import React from 'react';
import renderer from 'react-test-renderer';
import ArticleAboutEconomistLink from './article-about-economist-link';

describe('ArticleAboutEconomistLink', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<ArticleAboutEconomistLink />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
