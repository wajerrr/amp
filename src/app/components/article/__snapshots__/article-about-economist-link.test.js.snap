// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleAboutEconomistLink should match snapshot 1`] = `
<a
  className="css-fangro-StyledLink e11xntri0"
  href="https://www.economist.com/about-the-economist"
>
  <span
    className="css-tvpjd2-StyledIcon e11xntri1"
  >
    <svg
      version="1.1"
      viewBox="0 0 48 48"
      x="0px"
      xmlSpace="preserve"
      xmlns="http://www.w3.org/2000/svg"
      y="0px"
    >
      <clipPath
        id="a"
      >
        <path
          d="M598.9 770H401.6c-4 0-7.2-3.2-7.2-7.2V427.1c0-5-4-9-9-9H210.2c-4 0-7.2-3.2-7.2-7.2V237.2c0-4 3.2-7.2 7.2-7.2h579.6c4 0 7.2 3.2 7.2 7.2v173.7c0 4-3.2 7.2-7.2 7.2H613.3c-4 0-7.2 3.2-7.2 7.2v337.5c0 4-3.2 7.2-7.2 7.2z"
        />
      </clipPath>
      <path
        d="M44.722 40.183c0 2.634-2.117 4.772-4.726 4.772H7.862c-2.609 0-4.726-2.138-4.726-4.772V7.741c0-2.633 2.117-4.771 4.726-4.771h32.134c2.609 0 4.726 2.138 4.726 4.771v32.442z"
        fill="#fff"
      />
      <path
        d="M39.996.108H7.862C3.694.108.301 3.533.301 7.741v32.442c0 4.208 3.393 7.633 7.561 7.633h32.134c4.169 0 7.562-3.425 7.562-7.633V7.741c0-4.208-3.393-7.633-7.562-7.633zm4.726 40.075c0 2.634-2.117 4.772-4.726 4.772H7.862c-2.609 0-4.726-2.138-4.726-4.772V7.741c0-2.633 2.117-4.771 4.726-4.771h32.134c2.609 0 4.726 2.138 4.726 4.771v32.442z"
        fill="#191919"
      />
      <path
        d="M28.602 36.843h-9.323a.34.34 0 0 1-.34-.343V20.484a.425.425 0 0 0-.426-.429h-8.278a.34.34 0 0 1-.341-.344v-8.287a.34.34 0 0 1 .341-.343h27.389c.189 0 .341.153.341.343v8.287a.34.34 0 0 1-.341.344h-8.34a.34.34 0 0 0-.34.344V36.5a.341.341 0 0 1-.342.343z"
      />
      <g
        clipPath="url(#a)"
        transform="matrix(.04726 0 0 .0477 .3 .108)"
      >
        <path
          d="M-4271.8-3240.4h8850.6v7201.8h-8850.6z"
        />
        <clipPath
          id="b"
        >
          <path
            d="M-4271.8-3240.4h8850.6v7201.8h-8850.6z"
          />
        </clipPath>
        <path
          clipPath="url(#b)"
          d="M194 221h612v558H194z"
          fill="#191919"
        />
      </g>
    </svg>
  </span>
   About 
  <em>
    The Economist
  </em>
</a>
`;
